indico/htdocs/sass/partials/_jquery-indico-locationpicker.scss

```diff
@@ -0,0 +1,186 @@
+
+                input[type='text'] {
+                    width: 100%;
+                }
+            }
+
+            .typeahead-results {
+                position: absolute;
+                left: 0;
+
+                .typeahead-list {
+                    @include box-shadow(none);
+                    margin: 0;
+                    padding: 0;
+                    overflow: auto;
+                    max-height: 30em;
+                    width: 20em;
+                    position: relative;
+                    border-radius: 0;
+                    border-color: $gray;
+
+                    > li {
+                        background: white;
+                    }
+
+                    &.empty {
+                        border-width: 0px 0px 1px 0px;
+                        border-style: solid;
+                        border-color: $gray;
+
+                        li:first-child {
+                            display: none;
+                        }
+                    }
+                }
+
+                .just-use-option {
+                    display: none;
+                    cursor: pointer;
+                    background-color: white;
+                    text-align: center;
+                    padding: 6px 20px;
+                    border-width: 0px 1px 1px 1px;
+                    border-style: solid;
+                    border-color: $gray;
+                    font-style: italic;
+
+                    &:hover, &.active {
+                        background-color: #ebebeb;  // same as in typeahead
+                    }
+                }
+            }
+
+            &.result {
+                .i-location-input-field {
+                    z-index: 1050 !important;
+
+                    &.typeahead-hint {
+                        z-index: -1 !important;
+                        background-color: white !important;
+                    }
+                }
+
+                .just-use-option {
+                    display: block;
+                }
+
+                z-index: 1060 !important;
+            }
+        }
+
+        input.i-location-input-field {
+            width: 11.5em;
+            background-color: inherit;
+            padding-right: 2em !important;
+
+            &.disabled {
+                @extend %tilted-strips-bg;
+            }
+
+            &.bring-to-front {
+                background: white;
+            }
+
+            &:focus {
+              @include box-shadow($blue 0 0 5px);
+              @include border-all($color: $blue, $width: 1px);
+            }
+        }
+
+        .toggle-results-list {
+            position: absolute;
+            padding: 0.5em 0.6em;
+            top: 0;
+            right: 0;
+            line-height: 1.5;
+            cursor: pointer;
+        }
+
+        .location-inheritance-field {
+            height: 2.5em;
+
+            .checkbox-container {
+                vertical-align: middle;
+                line-height: 2.5;
```

Comment:
Yes! Since the font size is 1em, then why being explicit when specifying the line-height?
Isn't it better to say that line-height = 2.5 x font-size rather than 2.5em?